### ISSUE TITLE
Add channel image to RSS feed

### ISSUE THREAD

**Issue by sabesansathananthan** [forem/forem#9934]
**Is your feature request related to a problem? Please describe.**

In the dev.to RSS feed, I could not get my profile image

**Describe the solution you'd like**

medium.com gave the profile image link in their RSS feed. I could not get my dev.to profile image link in dev.to RSS feed

**Describe alternatives you've considered**

**Additional context**

**Comment by jonathanyeong** [comment]
Hi! Could I take this one?

**Comment by rhymes** [comment]
Thanks @jonathanyeong, I've assigned this to you!

**Comment by sabesansathananthan** [comment]
@rhymes @jonathanyeong <image > contains URL of user profile image. isn't it?

**Comment by rhymes** [comment]
@sabesansathananthan in this case yeah, if you're rendering the feed of the articles of a user. But you could be rendering an organization feed, thus you need the profile image of the org, or you could be rendering a tag feed where the `<image>` should be omitted and if it's the latest feed as well it shouldn't have an image tag.

See https://github.com/forem/forem/blob/9d384e1a933f498f21b96884cb821a3b3dac09f3/app/controllers/articles_controller.rb#L23 and https://github.com/forem/forem/blob/9d384e1a933f498f21b96884cb821a3b3dac09f3/config/routes.rb#L479-L482

 `<image>` is a composite element, the [spec says](https://cyber.harvard.edu/rss/rss.html#ltimagegtSubelementOfLtchannelgt):

![Screenshot_2020-10-04 RSS 2 0 Specification (RSS 2 0 at Harvard Law)](https://user-images.githubusercontent.com/146201/95011315-0c549a00-0630-11eb-82b5-f9dc41bbe403.png)

Basically something similar to this, copied from https://www.w3schools.com/XML/xml_rss.asp

```xml
<image>
  <url>https://www.w3schools.com/images/logo.gif</url>
  <title>W3Schools.com</title>
  <link>https://www.w3schools.com</link>
</image>
```

**Comment by sabesansathananthan** [comment]
@rhymes  yes I also expect that. <3

**Comment by jonathanyeong** [comment]
@rhymes that's a great tidbit thanks!

**Comment by jonathanyeong** [comment]
@rhymes I'm having some issues getting this setup with Docker. I've searched around the Forem issues but I can't find any that are related. Not sure if it's just my setup.

```
Step 27/31 : RUN RAILS_ENV=production NODE_ENV=production bundle exec rake assets:precompile
 ---> Running in 6cd038ffe2b1
rake aborted!
URI::BadURIError: both URI are relative
/opt/apps/forem/app/lib/url.rb:17:in `url'
/opt/apps/forem/config/initializers/imgproxy.rb:9:in `block in <main>'
/opt/apps/bundle/gems/imgproxy-1.2.0/lib/imgproxy.rb:30:in `configure'
```

I noticed that `base_url` is empty and `uri` equals `images`. I've also created a `.env` file. Am I missing something else?

**Comment by rhymes** [comment]
@jonathanyeong nope, that's a legit bug, but it should have been fixed by #10681 - can you make sure your branch is up to date with master? Does it still happen?

**Comment by jonathanyeong** [comment]
@rhymes doh, my branch wasn't up to date. Thank you! That resolved it.
